halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/aws/AwsAddAccountCommand.java

```diff
@@ -34,48 +35,70 @@ protected String getProviderName() {
   }
 
   @Parameter(
-      names = "--default-key-pair",
-      description = AwsCommandProperties.DEFAULT_KEY_PAIR_DESCRIPTION
+    names = "--default-key-pair",
+    description = AwsCommandProperties.DEFAULT_KEY_PAIR_DESCRIPTION
   )
   private String defaultKeyPair;
 
   @Parameter(
-      names = "--edda",
-      description = AwsCommandProperties.EDDA_DESCRIPTION
+    names = "--edda",
+    description = AwsCommandProperties.EDDA_DESCRIPTION
   )
   private String edda;
 
   @Parameter(
-      names = "--discovery",
-      description = AwsCommandProperties.DISCOVERY_DESCRIPTION
+    names = "--discovery",
+    description = AwsCommandProperties.DISCOVERY_DESCRIPTION
   )
   private String discovery;
 
   @Parameter(
-      names = "--account-id",
-      description = AwsCommandProperties.ACCOUNT_ID_DESCRIPTION
+    names = "--account-id",
+    description = AwsCommandProperties.ACCOUNT_ID_DESCRIPTION
   )
   private String accountId;
 
   @Parameter(
-      names = "--regions",
-      variableArity = true,
-      description = AwsCommandProperties.REGIONS_DESCRIPTION
+    names = "--regions",
+    variableArity = true,
+    description = AwsCommandProperties.REGIONS_DESCRIPTION
   )
   private List<String> regions = new ArrayList<>();
 
+  @Parameter(
+    names = {"--assume-role", "--role"},
+    description = AwsCommandProperties.ASSUME_ROLE_DESCRIPTION
+  )
+  private String assumeRole;
+
+  @Parameter(
+    names = {"--access-key-id", "--access-key"},
```

Comment:
It is, but it works a little bit too well... When true, JCommander no longer accepts the parameter at all, which makes scripting difficult. It displays an interactive prompt even if the parameter is provided.